mava/configs/system/rec_ippo.yaml

```diff
@@ -20,3 +20,7 @@ clip_eps: 0.2 # Clipping value for PPO updates and value function.
 ent_coef: 0.01 # Entropy regularisation term for loss function.
 vf_coef: 0.5 # Critic weight in
 max_grad_norm: 0.5 # Maximum norm of the gradients for a weight update.
+
+# --- Recurrent hyperparameters ---
+recurrent_chunk_size: ~ # The size of the chunks in which the recurrent sequences are divided during the training process.
```

Comment:
Is `None` a good default here. Like the comments but we should also set it to a good value

Reply:
Yes, we set it to None on purpose so that it recovers the current default Mava behaviour which is to have only a single `chunk` with the full rollout.